config/management/README.md

```diff
@@ -70,74 +70,89 @@ While this is compiled as a single binary it provides several different faciliti
-  association will share out the GitHub repository and a distinct name for each
-  participant.
-* Each participant must retrieve an appropriate
-  [token](https://github.com/settings/tokens) for their account that allows
-  access to the `repo` scope. This token should be stored locally on their disk
-  in a file accessible to the management application.
-* Each participant should initialize their respective key: `association`,
-  `owner`, or `operator` in a secure storage solution, likely Vault. And
-  prepare another token file accessible to the management application.
-* Each participant will upload their key to GitHub specifying their namespace
-  as their distinct name provided earlier:
-  * Association:
+The end-to-end process assumes that each participant has their own Vault
+solution and a token stored locally on their disk in a file accessible to the
+management tool.
+
+In addition, the association will provide a GitHub repository (and owner) along
+with a distinct namespace for each participant. GitHub namespaces equate to
+directories within the repository.
+
+Each participant must retrieve an appropriate GitHub
+[token](https://github.com/settings/tokens) for their account that allows
+access to the `repo` scope. This token must be stored locally on their disk in
+a file accessible to the management tool.
+
+Finally, each participant should initialize their respective key:
+`association`, `owner`, or `operator` in a secure storage solution. How this is
+done is outside the scope of this document.
+
+The remainder of this section specifies distinct behaviors for each role.
+
+### The Association
+
+* The association will publish a layout containing the distinct names and roles
+  of the participants, this is placed into a common namespace:
+```
+management set-layout \
+  --path PATH_TO_LAYOUT \
+  --backend 'backend=github;owner=OWNER;repostiory=REPOSITORY;token=PATH_TO_TOKEN;namespace=common'
+```
+* Each Member of the Association will upload their key to GitHub:
 ```
-management association-key \\
-    --local 'backend=vault;server=URL;token=PATH_TO_TOKEN' \\
+management association-key \
+    --local 'backend=vault;server=URL;token=PATH_TO_TOKEN' \
     --remote 'backend=github;owner=OWNER;repostiory=REPOSITORY;token=PATH_TO_TOKEN;namespace=NAME'
 ```
-  * Owner:
+
+### Validator Owners
+
+* Each Validator Owner member will upload their key to GitHub:
 ```
-management owner-key \\
-    --local 'backend=vault;server=URL;token=PATH_TO_TOKEN' \\
+management owner-key \
+    --local 'backend=vault;server=URL;token=PATH_TO_TOKEN' \
     --remote 'backend=github;owner=OWNER;repostiory=REPOSITORY;token=PATH_TO_TOKEN;namespace=NAME'
 ```
-  * Operator:
+
+### Validator Operators
+
+* Each Validator Operator member will upload their key to GitHub:
 ```
-management association-key \\
-    --local 'backend=vault;server=URL;token=PATH_TO_TOKEN' \\
+management operator-key \
+    --local 'backend=vault;server=URL;token=PATH_TO_TOKEN' \
```

Comment:
Replace URL with http://localhost:8200
PATH_TO_TOKEN with VAULT_TOKEN_FILE

Reply:
Is this only for demo purposes? because in practice this is deployment specific.

Reply:
Following my vault-init instructions, the vault server address will be http://localhost:8200. The token suggestion is to distinguish from the Github tokne.